docs/adrs/00-template.md

```diff
@@ -0,0 +1,31 @@
+# ADR 0000: [title]
```

Comment:
Let's document in contributor guide that we use ADRs, what they are, when something warrants an ADR, add'l reading material, etc.